Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -124,14 +143,26 @@ extension BrowserViewController {
             UIKeyCommand(input: "\\", modifierFlags: [.command, .shift], action: #selector(showTabTrayKeyCommand)), // Safari on macOS
             UIKeyCommand(input: "\t", modifierFlags: [.command, .alternate], action: #selector(showTabTrayKeyCommand), discoverabilityTitle: Strings.showTabTrayFromTabKeyCodeTitle)
         ]
+        
+        let bookmarkEditing = [
+            UIKeyCommand(input: "g", modifierFlags: [.command], action: #selector(showBookmarkCommand))
+        ]
+        
+        let findText = [
+            UIKeyCommand(input: "d", modifierFlags: [.command], action: #selector(findNextCommand)),
+            UIKeyCommand(input: "d", modifierFlags: [.command, .shift], action: #selector(findPreviousCommand))
+        ]
```

Comment:
Looks like these keys are swapped :) `d` should be for bookmark, `g` for find in page

Reply:
Shoot you are right, changing it.